src/microbe_stage/organelle_unlocks/UnlockProgress.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+/// <summary>
+///   Stores which organelles have been unlocked by the player.
+/// </summary>
+[UseThriveSerializer]
+public class UnlockProgress
+{
+    [JsonProperty]
+    public bool UnlockAll;
+
+    /// <summary>
+    ///   The organelles that the player can use.
+    /// </summary>
+    [JsonProperty]
+    private readonly HashSet<OrganelleDefinition> unlockedOrganelles =
+        SimulationParameters
+            .Instance
```

Comment:
The chop style shouldn't be preferred over the style of packing arguments. This is especially short in the grand scheme of things so I'd rewrite this as :
```
     SimulationParameters.Instance.GetAllOrganelles().Where(organelle => organelle.UnlockConditions == null).
     ToHashSet();
```

With the ToHashSet also on the same line if it fits.